src/CaptureClient/ApiEventProcessorTest.cpp

```diff
@@ -61,6 +61,7 @@ class MockCaptureListener : public CaptureListener {
               (uint64_t /*timestamp_ns*/,
                std::vector<orbit_grpc_protos::ModuleInfo> /*module_infos*/),
               (override));
+  MOCK_METHOD(void, OnPresentEvent, (const orbit_grpc_protos::PresentEvent&), (override));
```

Comment:
Shouldn't the call to this method be ensured by EXPECT_CALL in one of the tests?

Reply:
Yes good point, added a test in `CaptureEventProcessorTest.cpp`